### DISCUSSION TITLE
cant we anonymously play or download song in aur/linux client and have to mandatory login with spotify?

### DISCUSSION THREAD

**Discussion by Greatz08** [KRTirtho/spotube#1151] [Q&A]
is there no anonymous way to play and save and download music? no music loading and my net is working fine tried 3-4 times
![20240125_14h25m34s_grim](https://github.com/KRTirtho/spotube/assets/55040435/3247e4ad-8d38-46cb-b82f-d3f616551bc8)

**Comment by nepalivai** [comment]
No there no such way to play it without login! spotify key is necessary to use the app.

**Reply by Greatz08** [reply]
@nepalivai it was mentioned in main page that there is anonymous/guest login then why need for spotify key :-(( ?
